### PR TITLE
Add a litte more compat info for WPF

### PR DESCRIPTION
**Pull request by Thraka**

### CODE REVIEW

File: docs/desktop-wpf/migration/differences-from-net-framework.md [hunk 1]
```diff
@@ -8,10 +8,16 @@ ms.author: adegeo
 
 # Differences in .NET Core WPF
 
-This article describes the differences between .NET Framework's implementation of Windows Presentation Foundation (WPF) and .NET Core's implementation of WPF. .NET Core's implementation of WPF is based on the open-source release hosted on [GitHub](https://github.com/dotnet/wpf).
+This article describes the differences between Windows Presentation Foundation (WPF) on .NET Core and .NET Framework. WPF for .NET Core is an [open-source framework](https://github.com/dotnet/wpf) forked from the original WPF for .NET Framework source code.
+
+There are a few features of .NET Framework that .NET Core doesn't support. Meaning, WPF for .NET Core has a few limitations you should be aware of. For more information on unsupported technologies, see [.NET Framework technologies unavailable on .NET Core](../../core/porting/net-framework-tech-unavailable.md).
```
Comment (rpetrusha) [inline docs/desktop-wpf/migration/differences-from-net-framework.md]:
```suggestion
There are a few features of .NET Framework that .NET Core doesn't support. For more information on unsupported technologies, see [.NET Framework technologies unavailable on .NET Core](../../core/porting/net-framework-tech-unavailable.md).
```

File: docs/desktop-wpf/migration/differences-from-net-framework.md [hunk 2]
```diff
@@ -27,7 +33,7 @@ When your project uses `<PackageReference>`, packages aren't stored locally in a
 
 ## Code Access Security (CAS)
 
-Code Access Security (CAS) is not supported by .NET Core or WPF for .NET Core. All CAS-related functionality is treated under the assumption of full-trust. Therefore, WPF has been removing CAS-related code. The public API surface of these types still exists to ensure that calls into these types succeed. However, these calls don't actually do anything.
+Code Access Security (CAS) is not supported by .NET Core or WPF for .NET Core. All CAS-related functionality is treated under the assumption of full-trust. WPF for .NET Core has been removing CAS-related code. The public API surface of these types still exists to ensure that calls into these types succeed. However, these calls don't actually do anything.
```
Comment (rpetrusha) [inline docs/desktop-wpf/migration/differences-from-net-framework.md]:
```suggestion
Code Access Security (CAS) is not supported by .NET Core or WPF for .NET Core. All CAS-related functionality is treated under the assumption of full-trust. WPF for .NET Core removes CAS-related code. The public API surface of these types still exists to ensure that calls into these types succeed. However, these calls don't actually do anything.
```

**Review by rpetrusha — APPROVED** [review]
This looks good, @Thraka. I've left two suggestions for you to consider.

File: docs/desktop-wpf/migration/differences-from-net-framework.md [hunk 3]
```diff
@@ -50,6 +56,10 @@ Publicly defined CAS-related types were moved out of the WPF assemblies and into
...
 > - `AssemblyNameString`
 
+
+
 ## Next steps
 
-- [Learn how to port a .NET Framework WPF app to .NET Core.](convert-project-from-net-framework.md)
+- [Learn how to port a .NET Framework WPF app to .NET Core.][migrate-project]
+
+[migrate-project]: convert-project-from-net-framework.md
```
Comment (Thraka) [inline docs/desktop-wpf/migration/differences-from-net-framework.md]:
```suggestion

```

**Pull request merged**